FlowCrypt/Common UI/AttachmentManager.swift

```diff
@@ -45,24 +45,24 @@ final class AttachmentManager: NSObject {
 }
 
 extension AttachmentManager: AttachmentManagerType {
-    func open(_ attachment: MessageAttachment) {
-        filesManager.saveToFilesApp(file: attachment, from: self)
-            .sinkFuture(
-                receiveValue: {},
-                receiveError: { [weak self] error in
-                    self?.handle(error)
-                }
-            )
-            .store(in: &self.cancellable)
-    }
-
-    private func handle(_ error: Error) {
+    func download(_ file: FileType) {
         Task {
-            await controller?.showToast(
-                "\("message_attachment_saved_with_error".localized) \(error.localizedDescription)"
-            )
+            do {
+                let url = try await filesManager.save(file: file)
+                openDocumentsController(from: url)
+            } catch {
+                await controller?.showToast(
+                    "\("message_attachment_saved_with_error".localized) \(error.localizedDescription)"
+                )
```

Comment:
await not needed. good to follow warnings

<img width="1045" alt="image" src="https://user-images.githubusercontent.com/6306961/143503317-4c3b5d88-b415-4348-bcf5-3286515e203d.png">

Reply:
done